packages/maker/wix/src/Config.ts

```diff
@@ -1,137 +1,13 @@
-import { Features, MSICreator } from 'electron-wix-msi/lib/creator';
+import { MSICreator, MSICreatorOptions } from 'electron-wix-msi/lib/creator';
 
-export interface MakerWixConfig {
-  /**
-   * String to set as appUserModelId on the shortcut. If none is passed, it'll
-   * be set to com.squirrel.(Name).(exe), which should match the id given to
-   * your app by Squirrel.
-   */
-  appUserModelId?: string;
-  /**
-   * A comma separated string of extensions with each to be associated the app icon.
-   */
-  associateExtensions?: string;
-  /**
-   * The app's description
-   */
```

Comment:
What about keeping the jsdoc for defined properties?
To at least document that they have a predefined default value from the maker.

Reply:
Great suggestion: https://github.com/electron/forge/pull/3858/commits/ee1e39d0ad9b8ed4580986c8efe4ac0cd974d421